04.class/lib/memo.js

```diff
@@ -0,0 +1,95 @@
+import sqlite3 from "sqlite3";
+
+export class Memo {
+  static #db = null;
+
+  static get db() {
+    if (this.#db === null) {
```

Comment:
```suggestion
    if (!this.#db) {
```

でも十分な気がします。（16行目とかも）

undefinedとnullに関しては`===`で厳密に区別しない方がいいんじゃないかなーと思います。